### PR TITLE
Saxelrod hybridization to primary

### PR DESCRIPTION
**Pull request by shanaxel42**

### CODE REVIEW

File: notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py [hunk 1]
```diff
@@ -39,8 +39,10 @@
 # EPY: END markdown
 
 # EPY: START code
-exp = Experiment.from_json("https://dmf0bdeheu4zf.cloudfront.net/20180827/DARTFISH/experiment.json")
+
```
Comment (ttung) [inline notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py]: i'd drop the extra NL.

File: notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py [hunk 2]
```diff
@@ -39,8 +39,10 @@
...
+exp = Experiment.from_json('https://dmf0bdeheu4zf.cloudfront.net/20180828/DARTFISH/experiment.json')
 stack = exp.fov().primary_image
+
```
Comment (ttung) [inline notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py]: and here.

File: notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py [hunk 3]
```diff
@@ -66,7 +68,8 @@
 # EPY: END markdown
 
 # EPY: START code
-cnts_benchmark = pd.read_csv('https://dmf0bdeheu4zf.cloudfront.net/20180827/DARTFISH/fov_001/counts.csv')
+
```
Comment (ttung) [inline notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py]: and here.

**Comment by joshmoore** [comment]
For reference:

```
    Repository: spacetx/slicedimage
    Already up-to-date.

    Merged PRs:
      # PR 55 joshmoore 'Writer: set fov key in generate_parition_document'

    Repository: spacetx/sptx-format
    Already up-to-date.

    Merged PRs:
      # PR 11 ambrosejcarr 'RFC: add channel_map and version to codebook'
      # PR 12 ambrosejcarr 'ENH: hybridization_images -> primary_images'
      # PR 13 ambrosejcarr 'fix incorrect ID fields in codeword'
      # PR 14 ambrosejcarr 'DOC: add format diagram'
      # PR 21 joshmoore 'Validation output improvements'
      # PR 22 joshmoore 'Include json directories as packages'

    Repository: spacetx/starfish
    Updating 0a60530..58db7c0
    Previously merged:

    Merged PRs:
      # PR 412 ambrosejcarr 'DOC: Update readme with roadmap + support description'
      # PR 472 joshmoore 'Migrate docker to conda'
      # PR 494 shanaxel42 'Saxelrod hybridization to primary'
```


```
+ validate-sptx --experiment-json /tmp/experiment.json
...
Traceback (most recent call last):
  File "/home/starfish/.conda/envs/env/bin/validate-sptx", line 11, in <module>
    sys.exit(validate_sptx())
  File "/home/starfish/.conda/envs/env/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/starfish/.conda/envs/env/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/starfish/.conda/envs/env/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/starfish/.conda/envs/env/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/starfish/.conda/envs/env/lib/python3.6/site-packages/validate_sptx/validate_sptx.py", line 40, in validate_sptx
    with backend.read_contextmanager(experiment['hybridization_images']) as fh:
KeyError: 'hybridization_images'
```

**Pull request merged**